specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-05-01-preview/AutomationRules.json

```diff
@@ -912,19 +1048,56 @@
       }
     },
     "PlaybookActionProperties": {
+      "required": [
+        "logicAppResourceId"
```

Comment:
>logicAppResourceId [](http://example.com/codeflow?start=9&length=18)

Changing this property from optional to required requires breaking change review.

Reply:
I'm reverting this for now since this isn't part of the changes we intend to do in the new version.
I was told by the engineer that worked on the previous version (that added the endpoint that uses this definition) that it was always required but was missing in the swagger. I'll contact them offline to handle adding it in this version and the previous one